src/components/Widget/WelcomeWidget/WelcomeWidget.js

```diff
@@ -12,10 +13,11 @@ export default class WelcomeWidget extends Component {
 
   render () {
     const { group, settings = {} } = this.props
+    const message = settings.text || `We're happy you're here! Please take a moment to explore this page to see what's alive in our group. Introduce yourself by clicking Create to post a Discussion sharing who you are and what brings you to our group. And don't forget to fill out your profile - so likeminded new friends can connect with you!`
     return (
       <div styleName='welcome-widget'>
         <h2>{settings.title || `Welcome to ${group.name}!`}</h2>
-        <p>{settings.text || `We're happy you're here! Please take a moment to explore this page to see what's alive in our group. Introduce yourself by clicking Create to post a Discussion sharing who you are and what brings you to our group. And don't forget to fill out your profile - so likeminded new friends can connect with you!`}</p>
+        <p dangerouslySetInnerHTML={{ __html: markdown(message) }} />
```

Comment:
Are there alternatives to dangerouslySet, I thought they were pushing peeps away from using that?